packages/rancher-windows-exporter/charts/templates/daemonset.yaml

```diff
@@ -33,6 +33,7 @@ spec:
           mountPath: c:/scripts/
       - name: exporter-node
         image: {{ template "system_default_registry" . }}{{ .Values.clients.image.repository }}:{{ .Values.clients.image.tag }}
+        command: ["pwsh", "-f", "c:/scripts/run.ps1"]
```

Comment:
Since this step needs to be executed every time the container restarts and not just when the Pod is created, it's been moved directly into the script that gets executed.